test/e2e/specs/all-features-confirmation/footer/protractor.conf.js

```diff
@@ -0,0 +1,16 @@
+var pConfig = require('./../../../utils/protractor.configuration.js');
+
+var config = pConfig.getConfig({
+    configFileName: 'sample.json',
```

Comment:
Although `sample.json` will work fine, it should have it's own `dev.json` document like `footer.dev.json` (see the `dev.json` documents in `data_setup/config/record` for examples). Sample is meant to be just that, a sample. Part of creating this is also defining a separate `config.json` for the footer code that points to a schema that is *not* being used by any other specs in `all-features-confirmation`. Once you create this `config.json` for the footer configuration, make sure to add the configuration to `parallel-configs/all-features-confirmation.dev.json` so the config is loaded and the test runs along with the other parallel specs.